htmd/apps/acemd_v2.py

```diff
@@ -0,0 +1,405 @@
+                val = self.__dict__[cmd]
+                if cmd in self._outnames:
+                    val = self._outnames[cmd]
+                name = cmd
+                if cmd == 'scaling14':  # variables cannot start with numbers. We need to rename it here for acemd
+                    name = '1-4scaling'
+                text += '{name: <{maxwidth}}\t{val:}\n'.format(name=name, val=val, maxwidth=maxwidth)
+
+        if not quiet:
+            print(text)
+        else:
+            return text
+
+    def _writeBashRun(self, fname):
+        with open(fname, 'w') as f:
+            if self._version == 3:
+                f.write('#!/bin/bash\nacemd3 >log.txt 2>&1')
+            else:
+                f.write('#!/bin/bash\nacemd >log.txt 2>&1')
+        os.chmod(fname, 0o700)
+
+    def __repr__(self):
+        return self.show(quiet=True)
+
+    def writeConf(self, fname='input'):
+        """ Write an acemd configuration file
+
+        Parameters
+        ----------
+        fname : output file name
+        """
+        text = self.show(quiet=True)
+        fo = open(fname, 'w')
+        fo.write(text)
+        fo.close()
+
+
+if __name__ == "__main__":
+    # l=Acemd.protocols(quiet=True)
+    from htmd.home import home
+    from htmd.util import tempname
+    import filecmp
+    from glob import glob
+    import sys
+
+    tmpdir = tempname()
+
+    acemd = Acemd()
+    acemd.temperature = 350
+    acemd.restart = 'on'
+    acemd.restartfreq = 25
+    acemd.outputname = 'output'
+    acemd.xtcfile = 'myout.xtc'
+    acemd.xtcfreq = 500
+    acemd.timestep = 4
+    acemd.rigidbonds = 'all'
+    acemd.hydrogenscale = 0.3
+    acemd.switching = 'on'
+    acemd.switchdist = 9
+    acemd.cutoff = 5
+    acemd.exclude = 'scaled1-4'
+    acemd.scaling14 = 5.6
+    acemd.langevin = 'on'
+    acemd.langevintemp = 300
+    acemd.langevindamping = 0.8
+    acemd.pme = 'on'
+    acemd.pmegridspacing = 5
+    acemd.fullelectfrequency = 3
+    acemd.energyfreq = 10
+    acemd.constraints = 'on'
+    acemd.consref = '5dhfr_cube.pdb'
+    acemd.constraintscaling = 3.2
+    acemd.berendsenpressure = 'on'
+    acemd.berendsenpressuretarget = 14
+    acemd.berendsenpressurerelaxationtime = 2
+    acemd.tclforces = 'on'
+    acemd.minimize = 150
+    acemd.run = 555
+    acemd.celldimension = [3, 56, 2]
+    acemd.useconstantratio = 'on'
+    acemd.amber = 'off'
+    acemd.dielectric = 16
+    acemd.pairlistdist = 9
+    acemd.TCL = 'on'
+    acemd.bincoordinates = None
+    acemd.binvelocities = None
+    acemd.binindex = None
+    acemd.structure = '5dhfr_cube.psf'
+    acemd.parameters = 'par_all22_prot.inp'
+    acemd.extendedsystem = None
+    acemd.coordinates = '5dhfr_cube.pdb'
+    acemd.velocities = None
+    acemd.parmfile = None
+
+    acemd.coordinates = '5dhfr_cube.pdb'
+    acemd.setup(home(dataDir='dhfr/'), tmpdir, overwrite=True)
+
+    # Compare with reference
+    refdir = home(dataDir='test-acemd-v2')
```

Comment:
Where are these test-acemd-v2 references? Did you add them to the branch?

Reply:
I always forget that shit. Added now

Reply:
actually, how did this not fail on Travis?